doc/manual/source/threat-model.rst

```diff
@@ -0,0 +1,72 @@
+Threat Model
+============
+
+Based on the `threat model by restic <https://github.com/restic/restic/blob/master/doc/design.rst#threat-model>`_, we present the threat model for Routinator.
```

Comment:
If footnotes are supported, I would make the ref to restic a footnote. It's important to acknowledge inspiration, but it's a bit weird to put that front and center in the very first sentence.